bin/build_gfedb.py

```diff
@@ -625,6 +632,7 @@ def main():
             if hasattr(allele, 'seq'):
                 hla_name = allele.description.split(",")[0]
                 loc = allele.description.split(",")[0].split("*")[0]
+                print (hla_name, skip_alleles, loc)
```

Comment:
Is this needed ? Too much of `print` will just add more noise to logs.

Reply:
I will remove this print.  i agree.  Too messy.

Reply:
Any debug print info has been removed.